src/settings/AcquisitionUnits/AcquisitionUnitEditor/AcquisitionUnitEditorContainer.js

```diff
@@ -43,6 +43,7 @@ const AcquisitionUnitEditorContainer = ({ match, mutator, resources, close }) =>
         showCallout({
           messageId: `ui-acquisition-units.unit.actions.errors.save.${errorCode}`,
           type: 'error',
+          values,
```

Comment:
`values={ name: values.name }` - you don't need pass all form values

Reply:
updated